public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -192,9 +196,11 @@ export function GrafanaEvaluationBehaviorStep({
 
   const isGrafanaAlertingRule = isGrafanaAlertingRuleByType(type);
   const isGrafanaRecordingRule = isGrafanaRecordingRuleByType(type);
-  const { groupOptions, loading } = useFolderGroupOptions(folder?.uid ?? '', enableProvisionedGroups);
+  const { groupOptions, intervalsByGroup, loading } = useFolderGroupOptions(folder?.uid ?? '', enableProvisionedGroups);
   const [isEditingGroup, setIsEditingGroup] = useState(false);
 
+  // TODO This is way suboptimal, but it requires more changes to make it right, especiall in EditRuleGroupModal
+  // dispatch(fetchPromAndRulerRulesAction({ rulesSourceName: GRAFANA_RULES_SOURCE_NAME }));
```

Comment:
@gillesdemey PR https://github.com/grafana/grafana/pull/97490 might improve this one